### PR TITLE
libs/libxslt: Add at v1.1.37

### PR DESCRIPTION
**Pull request by der-eismann**
Still a bit unsure about:
- Putting docs in `-devel` or `-docs` package
- Putting python bindings in separate package

Input wanted @ReillyBrogan :slightly_smiling_face:

### CODE REVIEW

File: libs/libxslt/stone.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#
+name        : libxslt
+version     : '1.1.37'
+release     : 1
+homepage    : http://xmlsoft.org/libxslt/index.html
+upstreams   :
+    - https://download.gnome.org/sources/libxslt/1.1/libxslt-1.1.37.tar.xz : 3a4b27dc8027ccd6146725950336f1ec520928f320f144eb5fa7990ae6123ab4
+summary     : XSLT processor based on libxml2
+description : |
+    UPDATE DESCRIPTION
```
Comment (davidjharder) [inline libs/libxslt/stone.yml]: Needs a description

File: libs/libxslt/stone.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+upstreams   :
+    - https://download.gnome.org/sources/libxslt/1.1/libxslt-1.1.37.tar.xz : 3a4b27dc8027ccd6146725950336f1ec520928f320f144eb5fa7990ae6123ab4
+summary     : XSLT processor based on libxml2
+description : |
+    libxslt provides bindings to interact with XSLT files and
+    is based on GNOME's libxml2.
+license     :
+    - X11
+builddeps:
+    - pkgconfig(libxml-2.0)
```
Comment (ReillyBrogan) [inline libs/libxslt/stone.yml:19]: It looks like this supports being built against `xz` and `libgcrypt` (which we don't package yet but will sooner than later. Do you happen to know what building with those adds support for?

File: libs/libxslt/manifest.x86_64.jsonc [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+				"/usr/bin/xsltproc",
+				"/usr/lib/libexslt.so.0",
+				"/usr/lib/libexslt.so.0.8.20",
+				"/usr/lib/libxslt-plugins",
+				"/usr/lib/libxslt.so.1",
+				"/usr/lib/libxslt.so.1.1.37",
+				"/usr/lib/python3.11/site-packages/__pycache__/libxslt.cpython-311.opt-1.pyc",
+				"/usr/lib/python3.11/site-packages/__pycache__/libxslt.cpython-311.pyc",
+				"/usr/lib/python3.11/site-packages/libxslt.py",
+				"/usr/lib/python3.11/site-packages/libxsltmod.so",
```
Comment (ReillyBrogan) [inline libs/libxslt/manifest.x86_64.jsonc]: What do you think about splitting these out to a `python-libxslt` package?

Comment (der-eismann) [reply libs/libxslt/manifest.x86_64.jsonc]: Already saw that at the brotli package, I'm fine with that

File: libs/libxslt/stone.yml (hunk 2 above)
Comment (der-eismann) [reply libs/libxslt/stone.yml:19]: I couldn't find anything about xz in the GitLab repo. For crypto, `libexslt` seems to have some [encryption functions](https://gitlab.gnome.org/GNOME/libxslt/-/blob/master/libexslt/crypto.c).

Comment (ReillyBrogan) [reply libs/libxslt/stone.yml:19]: Yeah those look like they are no longer checked for.

File: libs/libxslt/monitoring.yml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+releases:
+  id: 13301
+security:
+  cpe: cpe:2.3:a:xmlsoft:libxslt
```
Comment (ReillyBrogan) [inline libs/libxslt/monitoring.yml]: Please adjust this to the v4 format.

**Review by ReillyBrogan — CHANGES_REQUESTED** [review]

File: libs/libxslt/monitoring.yml (hunk 4 above)
Comment (der-eismann) [reply libs/libxslt/monitoring.yml]: Done

**Comment by ReillyBrogan** [comment]
LGTM

**Comment by ikeycode** [comment]
lgtm

**Pull request merged**